includes/api/class-wc-admin-rest-reports-products-stats-controller.php

```diff
@@ -213,6 +220,28 @@ public function get_item_schema() {
 		return $this->add_additional_fields_schema( $schema );
 	}
 
+	/**	
+	 * Set the default results to 0 if API returns an empty array
+	 * 
+	 * @param $results mixed
```

Comment:
It looks like there is something wrong with this doc comments:
https://travis-ci.org/woocommerce/wc-admin/builds/443393089

Reply:
Whoops, thought I pushed this change already.  Should be okay now 😄